ehsm_kms_service/ehsm_kms_server.js

```diff
@@ -63,6 +64,8 @@ const server = (DB) => {
      */
     const { timer: nonce_cache_timer, nonce_database } = _nonce_cache_timer()
 
+    _token_time_verify(DB)

```

Comment:
will be called for all requests? 
should be called only for import key material?